### PR TITLE
feat!(i18n): replace `v-html` with `i18n` component

### PR DESCRIPTION
**Pull request by LuisMartinez15**
BREAKING CHANGE: Split `myHistory.message` translation into three different parts: `header`, `body` and `footer`.

EX-6928

### CODE REVIEW

File: src/i18n/messages/en.messages.json [hunk 1]
```diff
@@ -97,7 +101,7 @@
       "message": "No results found for <span class=\"x-font-bold x-text-auxiliary-50 \">'{query}'</span>. We show you results for"
```
Comment (annacv) [inline src/i18n/messages/en.messages.json:101]: `"message": "No results found for '{query}'. We show you results for"`

File: src/i18n/messages/en.messages.json [hunk 2]
```diff
@@ -97,7 +101,7 @@
...
     },
     "noResults": {
-      "message": "<span>Sorry but there are no results for <span class=\"x-font-bold \">\"{query}\"</span></span>You may be interested in these products"
+      "message": "Sorry but there are no results for {query}You may be interested in these products"
```
Comment (annacv) [inline src/i18n/messages/en.messages.json:104]:
```suggestion
      "message": "Sorry but there are no results for {query}. You may be interested in these products"
```

File: src/i18n/messages/es.messages.json [hunk 3]
```diff
@@ -91,10 +95,10 @@
       "addToCart": "AÑADIR AL CARRITO"
     },
     "spellcheck": {
-      "message": "No se han encontrado resultados para <span class=\"x-font-bold x-text-auxiliary-50 \">'{query}'</span>. Aquí tienes resultados para"
+      "message": "No se han encontrado resultados para '{query}'. Aquí tienes resultados para"
     },
     "noResults": {
-      "message": "<span>No se han encontrado resultados para <span class=\"x-font-bold \">\"{query}\"</span></span>Quizás te interesen estos productos"
+      "message": "No se han encontrado resultados para {query}Quizás te interesen estos productos"
```
Comment (annacv) [inline src/i18n/messages/es.messages.json:101]:
```suggestion
      "message": "No se han encontrado resultados para {query}. Quizás te interesen estos productos"
```

File: src/i18n/messages/fr.messages.json [hunk 4]
```diff
@@ -94,10 +98,10 @@
       "addToCart": "AJOUTER AU PANIER"
     },
     "spellcheck": {
-      "message": "Aucun résultat trouvé pour <span class=\"x-font-bold  x-text-auxiliary-50 \">'{query}'</span>. Nous vous montrons les résultats pour"
+      "message": "Aucun résultat trouvé pour '{query}'. Nous vous montrons les résultats pour"
     },
     "noResults": {
-      "message": "<span>Désolé mais il n'y a pas de résultats pour <span class=\"x-font-bold \">\"{query}\"</span></span>Vous pourriez être intéressé par ces produits"
+      "message": "Désolé mais il n'y a pas de résultats pour {query}Vous pourriez être intéressé par ces produits"
```
Comment (annacv) [inline src/i18n/messages/fr.messages.json:104]:
```suggestion
      "message": "Désolé mais il n'y a pas de résultats pour {query}. Vous pourriez être intéressé par ces produits"
```

**Review by annacv — CHANGES_REQUESTED** [review]
🚀 we will live better with this change

**Review by tiborux — DISMISSED** [review]

**Review by tiborux — APPROVED** [review]

**Review by annacv — APPROVED** [review]

**Pull request merged**